api/src/main/proto/stellarstation/api/v1/radio/radio.proto

```diff
@@ -51,11 +51,125 @@ message CommunicationProtocol {
   oneof Framing {
     // AX.25 protocol settings.
     AX25 ax25 = 1;
+
+    // CCSDS framing settings.
+    CCSDS ccsds = 2;
+
+    // Bit stream framing settings.
+    BitStream bit_stream = 3;
   }
 }
 
-// Communication of a stream of bits with no framing defined.
+// CCSDS framing https://public.ccsds.org/Pubs/131x0b3e1.pdf
+// It allows for a few common variations satellites use such
+// as setting a custom synchronization word, inverting the
+// convolutional coding polynomials, etc. The sequence of
+// error-correcting codes will be applied as they are defined
+// in the CCSDS standard.
+message CCSDS {
```

Comment:
Just to confirm, is CCSDS much more commonly used then space packet protocol? I just learned that CCSDS is an organization, not a protocol. But we should align with what's mostly used in practice.

Reply:
@rtubio Can you confirm that "CCSDS framing" is a common term (taking into account CCSDS is the name of an organization). I guess it's similar to IEEE, where we have protocols like IEEE 802.11x. Is there a number to attach to this one, or is it so common to just refer to it as CCSDS without any other specifier?

Reply:
From my experience so far, "CCSDS" is generally how people refer to this part of the standard, to distinguish it from AX.25, CSP, or Intelsat.  The hardware devices we are working with all call it "CCSDS".

Reply:
Btw, does CSP define framing and error correction? As far as I can tell, CSP (https://github.com/libcsp/libcsp) is also a network layer protocol that does not define synchronization or channel coding.

Reply:
I guess not.  It says it can be used on top of CCSDS, for example.

Reply:
The term for those message types at the link layer level should be "Transfer Frame", please check the following standard on page 2-4:

https://public.ccsds.org/Pubs/131x0b3e1.pdf

I also agree with @anuraaga that the term "transfer frame" sounds very generic and we might want to use something more specific. In that sense, something like "CCSDS Transfer Frames" might work as well. 

If these parameters are to be used within a CCSDS context (like, for example, when we are using Cortex or other systems), I think that using "transfer frames" is enough since people will already understand that it refers to the CCSDS link layers.

Reply:
@anuraaga should I rename this to `CCSDSTransferFrame` or `CCSDSTransferFrameSettings`? This is already in a `oneof` called `Framing` so I don't know whether or not this is too redundant.

Reply:
I think `CCSDSTransferFrame` is fine, the redundancy with `Framing` isn't such a big deal given people would read and understand the word "transfer frame". If we really wanted to reduce the redundancy, the alternative would be keeping as is but making sure to at least use the word transfer frame in all the comments.